src/config.js

```diff
@@ -0,0 +1,24 @@
+import LoginScreen from './screens/login/login';
+import ErrorScreen from './screens/error/error';
+import AdminHomeScreen from './screens/admin/admin';
+import Guards from './guards';
+
+export const routes = {
+  login: {
+    component: LoginScreen,
+    guards: [Guards.authGuard()],
```

Comment:
To fastest kick-start of the basic functionality we can temporary remove all guards from routes config.  
`guards:[]`

NOTE:
Do not forget left a TODO